cmd/peers.go

```diff
@@ -208,37 +211,61 @@ func (o *PeersOptions) Info() (err error) {
 
 // List shows a list of peers
 func (o *PeersOptions) List() (err error) {
+
+	// convert Page and PageSize to Limit and Offset
+	listParams := lib.NewListParams("", o.Page, o.PageSize)
+
 	if o.Network == "ipfs" {
 		res := []string{}
-		if err := o.PeerRequests.ConnectedIPFSPeers(&o.Limit, &res); err != nil {
+		if err := o.PeerRequests.ConnectedIPFSPeers(&listParams.Limit, &res); err != nil {
 			return err
 		}
 
 		for i, p := range res {
 			printSuccess(o.Out, "%d.\t%s", i+1, p)
 		}
 	} else {
-
 		// if we don't have an RPC client, assume we're not connected
-		if o.UsingRPC && !o.Cached {
+		if !o.UsingRPC && !o.Cached {
 			printInfo(o.Out, "qri not connected, listing cached peers")
 			o.Cached = true
 		}
 
 		p := &lib.PeerListParams{
-			Limit:  o.Limit,
-			Offset: o.Offset,
+			Limit:  listParams.Limit,
+			Offset: listParams.Offset,
 			Cached: o.Cached,
 		}
 		res := []*config.ProfilePod{}
 		if err = o.PeerRequests.List(p, &res); err != nil {
 			return err
 		}
 
+		if o.Cached {
```

Comment:
So I mentioned `less` in another thread, but after some digging, it looks like the best thing to do is to check the environment variable "PAGER", and use that, falling back to `less` otherwise: https://en.wikibooks.org/wiki/Guide_to_Unix/Environment_Variables#PAGER and https://unix.stackexchange.com/questions/213367/where-do-editor-pager-browser-environment-variables-come-from.

However, this also is POSIX specific, so it won't work on Windows. It's okay for now, but maybe add a TODO to fix this in the future so we know it's here.

Finally, do we always want to page this (outputting to less or whatever) even if there's only a small number of results? Maybe instead, check the length and just output to the terminal if it's small.

Reply:
dope this is great.

now we only print to a pager if we are looking at more than 25 items, we check the env variable PAGER, if it's not set we default to less.